sessionctx/variable/varsutil_test.go

```diff
@@ -467,24 +467,29 @@ func TestVarsutil(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, "10", val)
 	err = SetSessionSystemVar(v, TiDBStmtSummaryMaxStmtCount, "a")
-	require.Regexp(t, ".*Incorrect argument type to variable 'tidb_stmt_summary_max_stmt_count'", err)
+	require.Error(t, err)
```

Comment:
/cc @pingcap/tidb-configuration-reviewer

Reply:
@morgo Could you please take a look? There are only test changes.

Reply:
Sorry for the delay. LGTM, but you now have a conflict.

Reply:
Fixed.